lms/static/sass/course/_info.scss

```diff
@@ -56,6 +56,95 @@ div.info-wrapper {
       font-style: normal;
     }
 
+    div.upgrade-banner {
+      // This banner uses the Pattern Library's defined variables
+      background-color: $white;
+      border: 1px solid $uxpl-gray-background;
+      @include border-left(0px);
+      width: 100%;
+      display: table;
+
+      .green-border {
+        width: 10px;
+        height: 100%;
+        margin: 0;
+        vertical-align: top;
+        background: $uxpl-green-base;
+        display: table-cell;
+      }
+
+      .upgrade-icon {
+        vertical-align: top;
+        margin: $baseline;
+        display: inline-block;
+        img {
+          max-height: 50px;
```

Comment:
I don't see any other icons that are 50 pixels. Is this a new convention? It would be good to make this a variable.